cgi-bin/ljlib.pl

```diff
@@ -242,30 +250,6 @@ sub gearman_client {
     return $client;
 }
 
-sub mogclient {
-    return $LJ::MogileFS if $LJ::MogileFS;
```

Comment:
We can't remove LJ::mogclient - it's used in many, MANY other places that aren't touched here.